libs/editor/src/main/java/org/wordpress/android/editor/gutenberg/GutenbergContainerFragment.java

```diff
@@ -117,7 +115,7 @@ public void attachToContainer(ViewGroup viewGroup, OnMediaLibraryButtonListener
                     onGutenbergDidRequestEmbedFullscreenPreviewListener,
                     onGutenbergDidSendButtonPressedActionListener,
                     showSuggestionsUtil,
-                    onMediaFilesCollectionBasedBlockEditorListener,
+                    null, //todo
```

Comment:
This causes a crash issue, should collaborate with Sparrow

Reply:
In order to solve this, we need first to merge https://github.com/WordPress/gutenberg/pull/58140 and produce a new GBM version. @jarvislin I'll let you know when we have this ready. Thanks 🙇 !

Reply:
I presume that the code removal is not urgent, so I'll remove `onMediaFilesCollectionBasedBlockEditorListener` in a separate PR to merge disabling the Story block as soon as possible.

Reply:
@fluiddot No rush, take your time, thank you!

Reply:
**Update**
The Story block is disabled in the Gutenberg project, so it won't crash again.